impl/src/main/java/org/jboss/cdi/tck/tests/interceptors/invocation/InterceptorInvocationTest.java

```diff
@@ -83,32 +80,6 @@ public void testInitializerMethodsNotIntercepted() {
         assertTrue(AlmightyInterceptor.methodIntercepted);
     }
 
-    @Test
-    @SpecAssertion(section = BIZ_METHOD, id = "ia")
-    public void testProducerMethodsAreIntercepted() {
-
-        AlmightyInterceptor.reset();
-        getContextualReference(Wheat.class);
-
-        assertTrue(AlmightyInterceptor.methodIntercepted);
-    }
-
-    @Test
-    @SpecAssertion(section = BIZ_METHOD, id = "ic")
-    public void testDisposerMethodsAreIntercepted() {
-
-        AlmightyInterceptor.reset();
-
-        DependentInstance<Wheat> bean = newDependentInstance(Wheat.class);
-        Wheat instance = bean.get();
-        assertNotNull(instance);
-        AlmightyInterceptor.methodIntercepted = false;
-        bean.destroy();
-
-        assertTrue(WheatProducer.destroyed);
-        assertTrue(AlmightyInterceptor.methodIntercepted);
-    }
-
```

Comment:
Both these tests should IMHO stay out of `full`, they still apply to Lite I think?

Reply:
The WheatProducer in this test uses @Interceptors though, so it cannot be in lite.

Reply:
Ah I see, right. I wonder if we're losing coverage then. I think I'd rather change `WheatProducer` to use an interceptor binding, as producer/disposer interception is certainly part of Lite.